src/main/java/fi/otavanopisto/kuntaapi/server/integrations/management/ManagementPageIdMapEntityUpdater.java

```diff
@@ -165,7 +166,7 @@ private void updatePageIdMap(OrganizationId organizationId) {
     return result;
   }
 
-  private void applyMappings(Map<PageId, BaseId> result, boolean wildcard, BaseId parentId, PageId pageId) {
+  private void applyMappings(Map<BaseId, BaseId> result, boolean wildcard, BaseId parentId, PageId pageId) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused private "applyMappings" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)